### ISSUE TITLE
Certain emails cause an X Window System error

### ISSUE THREAD

**Issue by Djaler** [elementary/mail#196]
Juno. Just in a moment after opening Mail window it crashes with this output:
```
(pantheon-mail:14102): Gdk-ERROR **: 18:08:08.780: The program 'pantheon-mail' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadAlloc (insufficient resources for operation)'.
  (Details: serial 9514 error_code 11 request_code 53 (core protocol) minor_code 0)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the GDK_SYNCHRONIZE environment
   variable to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
```
This happen only when one specific letter is selected. You can see it in attachment
[original_msg.txt](https://github.com/elementary/mail/files/1811721/original_msg.txt)

**Comment by Djaler** [comment]
You can see this behaviour by the link https://youtu.be/3DgMX8a_cd8

**Comment by mdh34** [comment]
I can confirm this issue - certain emails give me the same error.
Backtrace: 
```
(pantheon-mail:17986): Gdk-ERROR **: 17:55:44.938: The program 'pantheon-mail' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadAlloc (insufficient resources for operation)'.
  (Details: serial 12480 error_code 11 request_code 53 (core protocol) minor_code 0)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the GDK_SYNCHRONIZE environment
   variable to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)

Thread 1 "pantheon-mail" received signal SIGTRAP, Trace/breakpoint trap.
0x00007ffff2b32c41 in ?? () from /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
(gdb) backtrace
#0  0x00007ffff2b32c41 in  () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#1  0x00007ffff2b35571 in g_log_writer_default () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#2  0x00007ffff2b337de in g_log_structured_array () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#3  0x00007ffff2b3422e in g_log_structured_standard () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#4  0x00007ffff3a46c31 in  () at /usr/lib/x86_64-linux-gnu/libgdk-3.so.0
#5  0x00007ffff3a53ab3 in  () at /usr/lib/x86_64-linux-gnu/libgdk-3.so.0
#6  0x00007fffea93bc8d in _XError () at /usr/lib/x86_64-linux-gnu/libX11.so.6
#7  0x00007fffea938bb7 in  () at /usr/lib/x86_64-linux-gnu/libX11.so.6
#8  0x00007fffea938c75 in  () at /usr/lib/x86_64-linux-gnu/libX11.so.6
#9  0x00007fffea939575 in _XEventsQueued () at /usr/lib/x86_64-linux-gnu/libX11.so.6
#10 0x00007fffea92b237 in XPending () at /usr/lib/x86_64-linux-gnu/libX11.so.6
#11 0x00007ffff3a4e08e in  () at /usr/lib/x86_64-linux-gnu/libgdk-3.so.0
#12 0x00007ffff2b2c998 in g_main_context_prepare () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#13 0x00007ffff2b2d36b in  () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#14 0x00007ffff2b2d54c in g_main_context_iteration () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#15 0x00007ffff685bd2d in g_application_run () at /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0
#16 0x00005555555ed3a7 in _vala_main (args=0x7fffffffdc08, args_length1=1) at ./src/client/application/main.vala:25
#17 0x00007ffff199bb97 in __libc_start_main (main=
    0x5555555cfb20 <main>, argc=1, argv=0x7fffffffdc08, init=<optimised out>, fini=<optimised out>, rtld_fini=<optimised out>, stack_end=0x7fffffffdbf8) at ../csu/libc-start.c:310
#18 0x00005555555cfb7a in _start ()

**Comment by mdh34** [comment]
Closed as a duplicate of #27
